### ISSUE TITLE
As a sys admin, I need to see a meaningful STT name and FIPS/TRIBE code in a user's profile

### ISSUE THREAD

**Issue by ADPennington** [raft-tech/TANF-app#1641]
**Description:**
After #1387 is merged, the STT field is no longer available in the user profile (See [below](#Supporting-Documentation)). Instead we have a `location_id` (which maps to the id in the STT model) which is not a meaningful value for staff responsible for ensuring that STT users (i.e. `data analysts`) can only submit data on behalf of their own STT. STT location and STT code are more meaningful for user management purposes.

These fields can be **read_only** at least until such a time as we iron out the **change** request flow. 

**Acceptance Criteria:** 

- [x] Allow saving of user STT with or without Role
- [x] STT_CODE code and needs to accept 3 digits rather than 2 (current). Leading zeros should be maintained (e.g. 078) (req removed from #217)
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

AC to be handled in separate PR
- [x] update the django admin user details page to make the relationship between a user and the STT they are associated with more obvious
   - dropdown select list for stt (if stt user, deactivated if regional user)
   - [STT Name](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/tdrs-backend/tdpservice/stts/management/commands/data/states.csv) field (e.g. Virginia) appears in user profile 
   - [STT_code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/tdrs-backend/tdpservice/stts/management/commands/data/states.csv) field (e.g. 11) appears as location ID in user profile
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**

- [ ] _design an updated mockup for the user profile_
- [ ] _add fields changes to user profile_ 
- [ ] change filtering of STT objects to be based on STT code

#### Supporting Documentation:

- _Current user profile fields as of v2.2.2_
![1641p1](https://user-images.githubusercontent.com/63075587/203368835-99d09d91-abd7-47c7-b50e-ed9c5a666348.PNG)

**Notes:**
- #217 blocks this because the `STT_CODE` field is not in the object
- Make new form validation permissive and easy to use

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_

**Comment by ADPennington** [comment]
drill scheduled for 4/4.

**Comment by stevenino** [comment]
@ADPennington to update screenshots.
